python/samples/demos/self_hosted_agent/README.md

```diff
@@ -0,0 +1,38 @@
+## Self Hosted Agents
+
+This sample demonstrates how to create multi-agent group chat application where agents are hosted outside of the Semantic Kernel. `agents` folder contains `fastapi` server that hosts 2 agents and exposes them via REST apis. `app` folder contains the Semantic Kernel application that orchestrates between the agents using `AgentGroupChat`.
+
+### Running the sample
+
+You will need to deploy Azure OpenAI models in [Azure AI Foundry](https://learn.microsoft.com/en-us/azure/ai-studio/). Keep the endpoint and the key handy.
+
+#### Running the server
+1. Navigate to `agents` folder.
+2. Create a virtual environment and install the dependencies.
+```bash
```

Comment:
If you add the main function to a `scripts` block in the pyproject.toml then you could make running much simpler: see https://docs.astral.sh/uv/concepts/projects/config/#entry-points